src/renderer/views/multiplayer/queue.vue

```diff
@@ -0,0 +1,64 @@
+<!--
+    Created by : knoxy5467
+
+    Intent: no need to write both a queue view for both casual and ranked, use the
+     same view for both, with some props for the differences
+
+    expected props: queueType: "casual" | "ranked" | "custom", queueName: string (for custom)
+
+-->
+<template>
+    <div>
+        <h1>{{ props.queueName }}</h1>
+        <Markdown
+            source="
+- Matchmaking queue toggles for 1v1, 2v2, 3v3, 4v4, 5v5, 6v6, 7v7, 8v8, Small FFA (3 - 5), Medium FFA (6 - 10), Large FFA (11 - 16), TeamFFA
+- Big shiny queue button, changes to cancel queue when queued already
+- Ready modal popup when match is found, shows status like 9/10 players ready
+- Should use TS and be hidden from everybody, even from yourself
+- Lower minimum balance deviation criteria (25%?), for faster queue times and because it's less important in casual
+"
+        />
+    </div>
+    <div>
+        <div class="flex-row flex-grow gap-md">
+            <span>X v X</span>
+            <Checkbox v-model="settings.queue1v1" label="1v1" />
+            <Checkbox v-model="settings.queue2v2" label="2v2" />
+            <Checkbox v-model="settings.queue3v3" label="3v3" />
+            <Checkbox v-model="settings.queue4v4" label="4v4" />
+            <Checkbox v-model="settings.queue5v5" label="5v5" />
+            <Checkbox v-model="settings.queue6v6" label="6v6" />
+            <Checkbox v-model="settings.queue7v7" label="7v7" />
+            <Checkbox v-model="settings.queue8v8" label="8v8" />
+        </div>
+        <div class="flex-row flex-grow gap-md">
+            <span>FFA</span>
+            <Checkbox v-model="settings.queueSmallFFA" label="Small FFA" />
+            <Checkbox v-model="settings.queueMediumFFA" label="Medium FFA" />
+            <Checkbox v-model="settings.queueLargeFFA" label="Large FFA" />
+            <Checkbox v-model="settings.queueTeamFFA" label="Team FFA" />
```

Comment:
Rather than hardcoding all the queues here, we'll want to be fetching them from the server and displaying them dynamically. I know that's not possible to do until that's implemented in the server, but I think it's still possible to write the frontend dynamically and just use some fake mock data for now to show the intention

Reply:
No problem, is the "queue" view component an idea you want to continue with?

Reply:
I was thinking more individual components for each queue, but typically I don't mind doing that later when it's more obvious which things should be split into their own components, so this is fine for now